Login_SpotifyUI/Login_SpotifyUI/View/SignUpView.swift

```diff
@@ -12,6 +12,8 @@ import TextFieldEffects
 class SignUpView: UIView {
     let identificationLabel = UILabel()
     let identificationTextField = YokoTextField()
+    let nickNameLabel = UILabel()
```

Comment:
```suggestion
    let nicknameLabel = UILabel()
```

아래 textField도 마찬가지 입니다